Makefile

```
@@ -123,6 +123,46 @@ test-coverage:
+# the canonical reseed. Idempotent — any stale backend on :8000 is killed
+# first. Out of scope: `stripe listen` (interactive auth; Stripe specs
+# self-skip). Don't run from worktrees: one backend instance, one :8000.
+e2e-setup:
+	@test -d $(BACKEND_DIR) || { echo "❌ $(BACKEND_DIR) not found — expected the backend checkout next to this repo"; exit 1; }
+	$(MAKE) -C $(BACKEND_DIR) e2e-seed run-e2e-daemon
+
+# Playwright suite with a mass-skip guard: journey specs self-skip when the
+# backend probe fails, so a wedged backend yields exit 0 with hundreds of
+# skips (false green). Warn loudly, don't fail — partial runs (--grep) skip
+# legitimately. `make test-e2e` remains the raw, guard-free run.
+e2e-run:
```

checkmake (0.3.2)

[warning] Target body for "e2e-run" exceeds allowed length of 5 lines (10).

(maxbodylength)